app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -33,7 +33,11 @@
         android:layout_margin="8dp"
         android:focusable="true"
         android:focusableInTouchMode="true"
-        android:cursorVisible="true"/>
+        android:cursorVisible="true"
+        android:maxLines="1"
```

Comment:
And what about the claim that "The attribute maxLines corresponds to the maximum height of the EditText, it controls the outer boundaries and not inner text lines."?
[http://stackoverflow.com/a/7094141/5369600]